docs/source/test_network.md

```diff
@@ -0,0 +1,578 @@
+You can find the `peer` binaries in the `bin` folder of the `fabric-samples`
+repository. Use the following command to add those binaries to your CLI Path:
+```
+export PATH=${PWD}/../bin:${PWD}:$PATH
+```
+You also need to set the `FABRIC_CFG_PATH` to point to the `core.yaml` file in
+the `fabric-samples` repository:
+```
+export FABRIC_CFG_PATH=$PWD/../config/
+```
+You can now set the environment variables that allow you to operate the `peer`
+ CLI as `org1.example.com`:
+```
+# Environment variables for org1.example.com
+
+export CORE_PEER_TLS_ENABLED=true
+export CORE_PEER_LOCALMSPID="Org1MSP"
+export CORE_PEER_TLS_ROOTCERT_FILE=${PWD}/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/ca.crt
+export CORE_PEER_MSPCONFIGPATH=${PWD}/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp
+export CORE_PEER_ADDRESS=localhost:7051
+```
+
+The `CORE_PEER_TLS_ROOTCERT_FILE` and `CORE_PEER_MSPCONFIGPATH` environment
+variables point to the org1 certificates in the `organizations` folder.
+
+If you used `./network.sh installCC` to install and start the fabcar chaincode,
+you can now query the ledger from your CLI. Run the following command
+to get the list of cars that were added to your channel ledger:
+```
+peer chaincode query -C mychannel -n fabcar -c '{"Args":["queryAllCars"]}'
+```
+
+If the command is successful, you can see the same list of cars that you saw in
+the logs when you ran the script:
+```
+[{"Key":"CAR0", "Record":{"make":"Toyota","model":"Prius","colour":"blue","owner":"Tomoko"}},{"Key":"CAR1", "Record":{"make":"Ford","model":"Mustang","colour":"red","owner":"Brad"}},{"Key":"CAR2", "Record":{"make":"Hyundai","model":"Tucson","colour":"green","owner":"Jin Soo"}},{"Key":"CAR3", "Record":{"make":"Volkswagen","model":"Passat","colour":"yellow","owner":"Max"}},{"Key":"CAR4", "Record":{"make":"Tesla","model":"S","colour":"black","owner":"Adriana"}},{"Key":"CAR5", "Record":{"make":"Peugeot","model":"205","colour":"purple","owner":"Michel"}},{"Key":"CAR6", "Record":{"make":"Chery","model":"S22L","colour":"white","owner":"Aarav"}},{"Key":"CAR7", "Record":{"make":"Fiat","model":"Punto","colour":"violet","owner":"Pari"}},{"Key":"CAR8", "Record":{"make":"Tata","model":"Nano","colour":"indigo","owner":"Valeria"}},{"Key":"CAR9", "Record":{"make":"Holden","model":"Barina","colour":"brown","owner":"Shotaro"}}]
+```
+
+If you want to invoke a chaincode to change an asset, rather than querying an
+asset, you also need to set the environment variables to communicate with the
+orderer:
+```
+# Environment variables for the orderer
+
+export CORE_PEER_LOCALMSPID="OrdererMSP"
+export CORE_PEER_TLS_ROOTCERT_FILE=${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem
+export CORE_PEER_MSPCONFIGPATH=${PWD}/organizations/ordererOrganizations/example.com/users/Admin@example.com/msp
+```
```

Comment:
Wouldn't these environment variables mean that you're acting AS the orderer org? Maybe I'm missing something.

Reply:
no, your right.